sdk/core/System.BinaryData/src/System.BinaryData.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>Contains the BinaryData which is useful for converting between string, stream, Json, and bytes.</Description>
+    <AssemblyTitle>System.BinaryData</AssemblyTitle>
+    <Version>1.0.0</Version>
```

Comment:
Are we planning to ship out of this repo? If not we should change the version to `-beta`

Reply:
We should ask the .NET folks what version they want for this library. If we are using our versioning scheme this should be 1.0.0-beta.1

Reply:
You are fine to use any version so long as it is less than 6.0.0.

Reply:
I assume we want to do one beta release so I will update to 1.0.0-beta.1

Reply:
Is a 1.0.0 to 6.0.0 jump going to feel unnatural?  Should we version it as if it shipped in the 3.1 wave?

That'd be 4.7.0 as the package version, and assembly version... uh... 4.0.0.0?

Reply:
I think it's disingenuous to do anything to "pretend" this is part of the .NET Core release wave when it is not.  I'd prefer something that's completely different so that folks don't draw incorrect conclusions.  We can explain the reason for the jump to 6.0.0: it's natural since the package is absorbed into our codebase/release at that time.

Reply:
> We can explain the reason for the jump to 6.0.0: it's natural since the package is absorbed into our codebase/release at that time.

I don't think that's how our customer think. When you consume a package in VS, you have no idea which source tree it comes from. IMHO, version numbers should have meaning regardless of the repo they were being built from.